solutions/04_clickhandler/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <script src="click.js"></script>
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script> 
```

Comment:
I think you need to load jQuery before your custom script (`click.js`) or else `click.js` will try to run before the `jQuery` constructor (`$`) exists